alfresco-dynamic-extensions-repo/build.gradle

```
@@ -4,13 +4,14 @@ subprojects {
     configurations {
         ampArtifact
         ampLib
-        compile.extendsFrom(ampLib)
+        compileOnly.extendsFrom(ampLib)
+        testCompile.extendsFrom(compileOnly)
```

I'm not sure if you want to do it like this, adding everything from `compileOnly` to the `testCompile` configuration, as this adds more dependencies to the test runtime classpath than you may need.

Do you need your BOM on the `testCompile` configturation, or is it enough to have
```suggestion
        testCompileOnly.extendsFrom(compileOnly)
```

---

I fully agree, but I followed the principles that are currently used in the rest of the project. 
(E.g. same `testCompile.extendsFrom(compileOnly)` in `annotations-runtime`.

I consider 'correcting' this out of scope...

What you suggest is ` testCompileOnly.extendsFrom(compileOnly)`
& explicitly importing the required `testRuntime` configurations?

---

Hmm, does testRuntime require anything from the alfresco BOM?

---

Apparently it does:
`* What went wrong:
Could not resolve all files for configuration ':alfresco-dynamic-extensions-repo:alfresco-dynamic-extensions-repo-50:testRuntimeClasspath'.
> Could not find org.alfresco:alfresco-repository:.
  Required by:
      project :alfresco-dynamic-extensions-repo:alfresco-dynamic-extensions-repo-50
> Could not find org.alfresco:alfresco-data-model:.
  Required by:
      project :alfresco-dynamic-extensions-repo:alfresco-dynamic-extensions-repo-50
`